docs/src/starknet/101.md

```diff
@@ -0,0 +1,205 @@
+# `sncast` 101
+
+This page showcases the standard usage of various `sncast` commands.
+For more in-depth explanation, please follow other guides from "sncast Overview".
+
+## Create and Deploy an Account
+
+First, create an account contract ready for deployment.
+This generates the account details but doesn't put it on the network yet.
+
+<!-- { "ignored": true } -->
+```shell
+$ sncast account create \
+  --name my_account \
+  --network sepolia
+```
+
+<details>
+<summary>Output:</summary>
+
+```shell
+command: account create
+add_profile: --add-profile flag was not set. No profile added to snfoundry.toml
+address: 0x[..]
+max_fee: [..]
+message: Account successfully created. Prefund generated address with at least <max_fee> STRK tokens. It is good to send more in the case of higher demand.
+
+After prefunding the address, run:
+sncast account deploy --network sepolia --name my_account
+```
+
+</details>
+
+After creating an account, send enough STRK tokens to the address to cover the price of the
+account creation.
+At least `max_fee` amount of fri (1 STRK = 10<sup>18</sup> fri) should be sent.
+
+> 💡 **Tip**
+> On Sepolia, [this free faucet](https://starknet-faucet.vercel.app/) can be used to fund the account.
+
+
+Then, deploy the account using the command that was provided in output, below `After prefunding the address, run`:
+
+<!-- { "ignored": true } -->
+```shell
+$ sncast account deploy \
+  --network sepolia
+  --name my_account
+```
+
+<details>
+<summary>Output:</summary>
+
+```shell
+command: account deploy
+transaction_hash: 0x[..]
+
+
+To see invocation details, visit:
+transaction: https://sepolia.starkscan.co/tx/0x[..]
+```
+
+</details>
+
+Read more about accounts [here](./account-import.md)
+
+> 💡 **Tip**
+> Existing accounts can be imported to `sncast` as well. See [Importing Accounts](./account-import.md) for more
+> information.
+
+## Calling a Contract
+
+Calling a contract doesn't require an account
+
+```shell
+$ sncast call \
+  --contract-address 0x04718f5a0fc34cc1af16a1cdee98ffb20c31f5cd61d6ab07201858f4287c938d \
+  --function name \
+  --network sepolia
+```
+
+<details>
+<summary>Output:</summary>
+
+```shell
+command: call
+response: [0x537461726b6e657420546f6b656e]
+```
+
+</details>
+
+[Read more about calling contracts here](./call.md)
+
+## Sending a Transaction
+
+To send a transaction, invoke a contract.
+
+> 📝 **Note**
+> `--account` argument must be passed before invoke subcommand.
+> This is the same for other commands that send transactions as well.
+
+```shell
+$ sncast \
+  --account my_account \
+  invoke \
+  --contract-address 0x04718f5a0fc34cc1af16a1cdee98ffb20c31f5cd61d6ab07201858f4287c938d \
+  --function transfer \
```

Comment:
In other docs examples for invoke we use double quotes, check if they aren't needed.

Reply:
Don't seem so, at least for these simple function names